### PR TITLE
Make the repo more accessible

### PR DESCRIPTION
**Pull request by theTisch21**
This PR:
- Adds a start script for linux `yarn run start:linux`
- Edits the README.md to be a bit more readable by placing important information above the huge list of dependencies 
- Edits the README.md to make it clear that the screenshots action does not work for pull requests from outside the repo
- Edits CONTRIBUTING.md, adding instructions for how to run the website locally
- Edits the footer, making it clear that the website is open source

Edit: Sorry about the commit message being cutoff, I didn't realize it hit the character limit.

### CODE REVIEW

**Comment by daniel-sudz** [comment]
Screen Shot 2021-05-01 at 9 14 19 PM
Screen Shot 2021-05-01 at 9 14 35 PM

@theTisch21 I'm not exactly sure why but the spacing is slightly off now (something about the <p> and <a> blocks is causing this although it seems to be the correct way to do it)

File: website/src/Components/Footer/footer.tsx [hunk 1]
```diff
@@ -4,12 +4,16 @@ const Footer = () => {
...
-      <a className="text-white" href="mailto:eastlakekey@gmail.com">
-        Issues/Suggestions: eastlakekey@gmail.com
-      </a>
-      <a className="text-white pb-2" href="https://repo.ehs-service.org">
-        About: repo.ehs-service.org
-      </a>
+      <p className="text-white pb-2">
+          This tracker is open source: <a href="https://github.com/eastlakehs/community-service-tracker">
+          https://github.com/eastlakehs/community-service-tracker
+        </a>
```
Comment (daniel-sudz) [inline website/src/Components/Footer/footer.tsx:10]:
```suggestion
      <p className="text-white pb-2">
          This tracker is open source: <a href="https://github.com/eastlakehs/community-service-tracker">
          contribute here
        </a>
```

I think it's cleaner to not leave in the full url

**Review by daniel-sudz — CHANGES_REQUESTED** [review]

File: website/src/Components/Footer/footer.tsx (hunk 1 above)
Comment (daniel-sudz) [reply website/src/Components/Footer/footer.tsx:10]: I've also changed the redirect for ```repo.ehs-service.org``` back to this repo so you should be able to use it correctly now if you want to (as always it takes about 24 hours to cache-bust and you might to flush dns locally).

File: README.md [hunk 2]
```diff
@@ -21,14 +21,31 @@
    <img src="./Docs/images/wolflogo.png" width="200" height="200" />
 </p>
 
-# Status: Currently in beta
+# Status: Live at <a href="https://ehs-service.org"> http://ehs-service.org </a>
+
+# Please see CONTRIBUTING.md for how to run the website locally
```
Comment (Zjjc123) [inline README.md:26]: Could do something like

[CONTRIBUTING.md](https://github.com/eastlakehs/community-service-tracker/blob/master/CONTRIBUTING.md)

`[CONTRIBUTING.md](https://github.com/eastlakehs/community-service-tracker/blob/master/CONTRIBUTING.md)`

**Review by Zjjc123 — COMMENTED** [review]
I like it though! great changes!

File: README.md (hunk 2 above)
Comment (theTisch21) [reply README.md]: Done!

File: website/src/Components/Footer/footer.tsx (hunk 1 above)
Comment (theTisch21) [reply website/src/Components/Footer/footer.tsx]: Your changes are merged. Do you want me to change it back to `repo.ehs-service.org`?

Comment (daniel-sudz) [reply website/src/Components/Footer/footer.tsx]: @theTisch21 yeah let's move it back. The DNS is resolving correctly for me now. The original also has the spacing correct (see the problem above that I commented with the footer)

**Review by Zjjc123 — DISMISSED** [review]

**Review by Zjjc123 — APPROVED** [review]

**Comment by Zjjc123** [comment]
@daniel-sudz I approved for my requested changes or should I not do that until everything is fixed

**Comment by daniel-sudz** [comment]
@Zjjc123 it's fine -- I'm just trying to fix the bottom spacing problem

**Review by daniel-sudz — APPROVED** [review]
Screen Shot 2021-05-02 at 12 18 56 AM

I fixed the spacing problem myself

**Pull request merged**